src/module/ngx-auth-firebase-u-i.module.ts

```diff
@@ -41,6 +41,8 @@ export {LegalityDialogComponent} from './components/legality-dialog/legality-dia
 export {AuthProcessService, AuthProvider} from './services/auth-process.service';
 export {FirestoreSyncService} from './services/firestore-sync.service';
 
+// guards
+export {LoggedInGuard} from './guards/logged-in.guard';
```

Comment:
awesome! the guard can now be imported from `ngx-auth-firebaseui`'s module!

However, we need to provide it in `NgxAuthFirebaseUIModule`

similar like below...

```ts
export class NgxAuthFirebaseUIModule {
  static forRoot(configFactory: FirebaseAppConfig,
                 appNameFactory?: () => string,
                 config: NgxAuthFirebaseUIConfig = defaultAuthFirebaseUIConfig): ModuleWithProviders {
    return {
      ngModule: NgxAuthFirebaseUIModule,
      providers:
        [
          {
            provide: FirebaseOptionsToken,
            useValue: configFactory
          },
          {
            provide: FirebaseNameOrConfigToken,
            useFactory: appNameFactory
          },
          {
            provide: NgxAuthFirebaseUIConfigToken,
            useValue: config
          },
          AuthProcessService,
          FirestoreSyncService,
          LoggedInGuard // the guard is now within the maim module provided and can be used
        ],
    };
  }
}
```